src/rulesets/tests/__snapshots__/end-end.test.ts.snap

```diff
@@ -64137,6 +64219,40 @@ Array [
     },
     "where": "added field: strangeness",
   },
+  Object {
+    "change": null,
+    "condition": "be provided for every resource document",
```

Comment:
@cmars  -- looks like some of the examples are failing now because stability is missing